libvast/vast/command.hpp

```diff
@@ -95,6 +95,14 @@ caf::message run(const command& cmd, caf::actor_system& sys,
 caf::message run(const command& cmd, caf::actor_system& sys,
                  caf::settings& options, const std::vector<std::string>& args);
 
+/// Gets a subcommand from its full name.
+const command* resolve(const command& cmd,
+                       std::vector<std::string_view>::iterator position,
+                       std::vector<std::string_view>::iterator end);
+
+/// Gets a subcommand from its full name.
+const command* resolve(const command& cmd, std::string_view name);
+
```

Comment:
The documentation needs a bit of polishing: arguments and return value should be documented.